tools/triage_stats.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# pylint: disable=invalid-name,bare-except
+
+# Script to output table with the count of triage related tickets, grouped by week for the past 3 weeks
+
+import json
+import tabulate
+import jira_cmd
+from collections import defaultdict
+
+
+class TriageStats:
+    def __init__(self):
+        parser = jira_cmd.build_parser()
+        def my_exit(msg):
+            pass
+        parser.error = my_exit
+        self.args = parser.parse_args()
+        self.args.print_json = True
+        self.args.linked_issues = True
+        self.stats = {}
+
+    def add_past_week(self, past_week_num):
+        self.args.search_query = 'project = MGMT AND component = "Assisted-installer Triage" AND labels in (AI_CLOUD_TRIAGE) AND created < -{}w AND created >= -{}w'.format(past_week_num-1, past_week_num)
+        if past_week_num == 1:
+            self.args.search_query = 'project = MGMT AND component = "Assisted-installer Triage" AND labels in (AI_CLOUD_TRIAGE) AND created >= -1w'
+
+        print(self.args.search_query)
+        j = jira_cmd.main(self.args)
+        data = json.loads(j)
+        for i in data:
+            if i['key'] not in self.stats:
+                self.stats[i['key']] = dict(key=i['key'], summary=i['summary'], status=i['status'])
+            self.stats[i['key']]['week -{}'.format(past_week_num)] = i['count']
+
+
+a = TriageStats()
```

Comment:
Let's guard it inside ``if __name__ == "__main__":``

Reply:
ok